app/queries/Connect.rb

```diff
@@ -0,0 +1,39 @@
+class Connect
+  def initialize(user1, user2)
+    @user1 = user1
+    @user2 = user2
+  end
+
+  def chat
+    @chatroom = Chatroom.create

```

Comment:
A chatroom always needs to have two users, so this command is supposed to fail when you have fixed the validations.
Either create it by calling Chatroom.new and then add the users manually, or directly include the users in the create call